go.mod

```diff
@@ -24,19 +24,18 @@ require (
 	github.com/decred/dcrd/wire v1.4.0
 	github.com/decred/go-socks v1.1.0
 	github.com/decred/slog v1.1.0
-	github.com/ethereum/go-ethereum v1.9.25
+	github.com/ethereum/go-ethereum v1.10.4
```

Comment:
Aaaand they released 1.10.5 last night :laughing: 
https://github.com/ethereum/go-ethereum/releases/tag/v1.10.5

Reply:
updating...

Reply:
And wouldn't you know it broke something.

Reply:
Screw it.  Can be in another PR.  I don't want to hold you up more